resources/js/components/Board.js

```diff
@@ -126,10 +126,10 @@ export class Board extends React.Component {
   }
   //_onHoldStart and _onHoldEnd borrowed from: https://www.youtube.com/watch?v=A95mIE2HdcY
   _onHoldStart = (tileType, tileId, tileX, tileY) => {
-      if (this.props.board_id === 1) {
-        //return;
+      if (this.props.board_id === '1') {
```

Comment:
I didn't know that the `board_id` prop gets passed as a string, that's annoying. Just to be safe, we can loosen up the type checking with
```suggestion
      if (this.props.board_id == '1') {
```
so it catches both `1` and `'1'`